src/components/Filter/Filter.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Filter from '../Filter/Filter';
+
+describe('Filter component', () => {
+  const props = {
+    value: 'aaa',
+    onChange: () => jest.fn(),
```

Comment:
just `jest.fn()` is enough

Reply:
`onChange ` value has been updated to onChangeMock and before this `const onChangeMock` with `jest.fn()` value has been added.